apps/cli/.eslintrc.yaml

```diff
@@ -0,0 +1,13 @@
+extends:
+  - eslint:recommended
```

Comment:
I was seeing errors like this
<img width="634" alt="image" src="https://github.com/onmetal-dev/metal/assets/72655/fe7694d9-0a73-48e8-a135-c78b2539215d">

AI told me the answer was to add 
```suggestion
  - eslint:recommended
  - plugin:@typescript-eslint/recommended
```
and it went away